string target_frame
---
geometry_msgs/PoseArray circles
bool success